### DISCUSSION TITLE
Contact form is broken

### DISCUSSION THREAD

**Discussion by camsong** [npm/feedback#688] [Website] [answered]
I need to raise a help, after I filled the form, it always show 'error loading page'.

![image](https://user-images.githubusercontent.com/948896/168743170-765396a8-8795-4ac4-8a69-089eee122e5b.png)

Filled the form, and click 'Submit', it will show an error as follows
![image](https://user-images.githubusercontent.com/948896/168743214-2ee9794b-26c0-460b-a089-b773347ed18f.png)

**Comment by MylesBorins** [comment, marked as answer]
Thanks for sharing. I'm surfacing this to engineering to figure out what is going on.

**Reply by NING0121** [reply]
Sorry, I may need your help. I found that it still has this problem occasionally. Did I submit it too often?
![image](https://github.com/npm/feedback/assets/61087497/d3ffeaaa-05a0-4ecf-b50f-0e38012ca0b5)
@MylesBorins

**Comment by camsong** [comment]
It works now, thanks for your response @MylesBorins
